go/vt/vttablet/tabletserver/tx/api.go

```diff
@@ -181,3 +204,9 @@ func (p *Properties) String(sanitize bool, parser *sqlparser.Parser) string {
 		printQueries(),
 	)
 }
+
+func (p *Properties) GetQueries() []Query {
```

Comment:
the other methods are safe against `p` being nil, but this method is not. is that intentional?